src/js/components/Select/__tests__/SelectUncontrolled-test.js

```diff
@@ -10,10 +13,20 @@ import { Grommet } from '../..';
 import { Select } from '..';
 
 describe('Select', () => {
+  window.scrollTo = jest.fn();
   beforeEach(createPortal);
-
   afterEach(cleanup);
 
+  test('should not have accessibility violations', async () => {
+    const { container } = render(
+      <Grommet>
+        <Select options={['one', 'two', 'three']} a11yTitle="test" />
```

Comment:
we should add `a11yTitle` on the 'basic'  test below as well.

Reply:
Included in the most recent push